Drivers/Strength_WOrkout/Strength_Workout.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python3
+#! bin/bash
+#! bin/sh
+import sys
+import time
+import os
+from mqtt_client import MQTTClient
+from StrengthWorkout_class import StrengthWorkout
+from dotenv import load_dotenv, set_key
+
+MAX_WORKOUT_DURATION = 20  # Maximum duration of the workout in minutes
+
+def perform_strength_workout(strength_workout_object):
+    print("Starting strength workout in 5 seconds...")
+    time.sleep(5)
+
+    start_time = time.time()
+    try:
+        while True:
+            current_time = time.time() - start_time
+            if current_time >= (strength_workout_object.duration * 60) or current_time >= (MAX_WORKOUT_DURATION * 60):
+                break
+
+            # Read the user's desired resistance percentage from the command line
+            resistance_level = input("Enter the resistance percentage (0-100%): ")
+            try:
+                resistance_level = float(resistance_level)
+                if resistance_level < 0 or resistance_level > 100:
+                    print("Invalid resistance percentage. Please enter a value between 0 and 100.")
+                    continue
+
+
+                # Store the resistance level, resistance percentage, and current time in the strength workout object
+                strength_workout_object.resistance_data.append(resistance_level)
+                strength_workout_object.time_data.append(current_time)
+
+                # Perform the strength workout action based on the resistance level
+                strength_workout_object.perform_action(resistance_level)
```

Comment:
This method needs to be implemented in the StrengthWorkout class:

THis method should publish the parameter of the resistance parameter to the resistance topic.

include this method as part of this program, not as a method call from the strength workout object.

def perform actions(int resistence_level):
    mqtt_client.publish(f'bike/{deviceId}/resistence), resistence_level)